src/components/atoms/BaseCard/BaseCard.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState, KeyboardEvent } from 'react';
+import { createUseStyles } from 'react-jss';
+
+/**
+ * Properties for BaseCard component
+ */
+export interface BaseCardProps {
+  /** Element to show at card front face */
+  frontFace?: JSX.Element;
+  /** Element to show at card back face */
+  backFace?: JSX.Element;
+  /** Defines the card height */
+  height?: number;
+  /** Defines the card width */
+  width?: number;
+}
+
+const useStyles = createUseStyles({
+  // Generic classes
+  cardFace: {
+    backfaceVisibility: 'hidden',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    color: 'cornSilk',
+    textAlign: 'center',
+    font: `3em/240px 'Helvetica Neue', Helvetica, sans-serif`,
+    boxShadow: '-5px 5px 5px #aaa',
+    transition: '0.6s',
+  },
+  // Modal classes
+  backVisible: {},
+  frontVisible: {},
+  // Style classes
+  card: ({ width, height }) => ({
+    width: width || 600,
+    height: height || 400,
+    perspective: 1000,
+    cursor: 'pointer',
+    '&$backVisible $cardFlipper': { transform: 'rotateY(180deg)' },
+    '&$backVisible $cardFront': { opacity: 0 },
+    '&$backVisible $cardBack': { opacity: 1 },
+    '&$frontVisible $cardFront': { opacity: 1 },
+    '&$frontVisible $cardBack': { opacity: 0 },
```

Comment:
I don't really like this part, feels kind of duplicated, can't we simplify it?

Reply:
I don't like either!
But to be honest, I have tried to find a way of fixing it at react-jss but no clue..

It's kind of weird, I am still trying to find a way that pleases me. What do you think about:
```typescript
'&$backVisible $cardFront, &$frontVisible $cardBack': { opacity: 0 },
'&$backVisible $cardBack, &$frontVisible $cardFront': { opacity: 1 },
```
?